### PR TITLE
Meet the team

### LINKED ISSUE #33: Meet the Team (2 weeks)
WEEK 1: DESKTOP
WEEK 2: MOBILE 

Note: 

1.  When developing on desktop, add the styling with consideration for what would be easiest to add media queries for on mobile. 
2. The desktop design applies to > tablet and mobile applies to < tablet
3. Until you get the actual headshot images from the headshot day, feel free to just use the same image for everyone or some kind of placeholder since we're gonna need to swap it out anyways. After the headshot day, take initiative to reach out to whoever took the photos to request them right away so you can add them to your PR. 

- [ ] Develop this section according to the design on desktop + mobile (responsive all sizes)
- [ ] Create a new component, MeetTheTeamSection in main-site and use it in the index.ts of main-site
- [ ] For the mobile version, use the Arrow component for the arrows and use the getLeftOrRight function in utils.ts to get the left or right item (see about section implementation as a reference). 
Screen Shot 2022-10-02 at 8 15 51 PM
Screen Shot 2022-10-02 at 8 16 19 PM

### PR DESCRIPTION
**Pull request by jhylisa666**
Meet the team section

Week2: 
- made make-the-team responsive 
- made make-the-team the mobile design on mobile screen
- added mobile-team-column folder

https://user-images.githubusercontent.com/92692037/197645239-b03425d8-34ab-47f0-8186-5d23596f6c3d.mov

Week1: 

Fixes #33

Changelist:

- Made make-the-team section
- created new component make-the-team
- created component folder team-column within make-the-team
- created JSON data for each column

Notes:

- already reached out to Mike and David for real head shots

Screenshots & Screencasts:

https://user-images.githubusercontent.com/92692037/196317696-cf2ced4a-6da5-4df1-8e43-021816566b73.mov

### CODE REVIEW

File: main-site/components/meet-the-team/MeetTheTeamSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/meet-the-team/MeetTheTeamSection.styles.ts]: change to padding: 10em 0;

File: main-site/components/meet-the-team/team-column/TeamColumn.tsx
Comment (jzhzhang66) [inline main-site/components/meet-the-team/team-column/TeamColumn.tsx]: For the StyledLabel, when its 2 lines, it shifts all the images in the column down. 
Screen Shot 2022-10-18 at 3 20 03 PM

Can we always keep this as two lines by setting the width or max-width and then add margin: -0.4em  ? This will shift the images below it up so it can match the other columns

You can pass in a prop to your styled label that represents if ur thing is two lines or not and then apply the margin according to that

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Lisa this looks great!!! Awesome job and thanks for taking initiative to reach out to david + mike!!<3

I left a few comments and also, from the tablet -> tabletLg sizing, could we reduce the image size? They get too big to fit on the screen comfortably 

https://user-images.githubusercontent.com/59738880/196524861-90d4c0b1-9336-4b33-ae6d-5c7f2087c7a1.mov

You might also need to reduce the column width or font size bc the titles don't fit either
Screen Shot 2022-10-18 at 3 24 22 PM

**Review by jzhzhang66 — COMMENTED** [review]
I noticed the judy-placeholder is an svg - for images let's use png 

svg is for vectorized objects (things that our design team creates, etc.) but if we have a bunch of images that are exported as svg I worry that it will slow down the site

**Review by mike10911 — CHANGES_REQUESTED** [review]
Hey Lisa!!! This looks so fire! One small design change!

In the figma, both the heading and sub-heading text font seem to be larger.
Screen Shot 2022-10-19 at 3 33 52 AM
Screen Shot 2022-10-19 at 3 33 30 AM
I believe "meet the team" should have the same size font as the other section headings

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
lookin good!! 2 more small changes - 

- for the two line ones could you please add a tiny bit of padding-left to center the text a bit more? Screen Shot 2022-10-20 at 8 35 03 AM
- The header is still not the same exact font as the other headers (check what Faq or about section is using I think it's H2). Also I realized The calendar section is also not the correct title  size could you please change that too?

File: main-site/components/events-calendar-section/EventsCalendarSection.styles.ts
Comment (jzhzhang66) [inline main-site/components/events-calendar-section/EventsCalendarSection.styles.ts]: I think this centered ur entire section

It looked like this before
Screen Shot 2022-10-24 at 11 33 57 PM

Now it looks like this
Screen Shot 2022-10-24 at 11 36 41 PM

Make sure you're only centering the section title here!

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Hey kween!!! Awesome job on getting the logic to work!!!

The main requested change is that on the Figma, this should be the layout all the time for screensizes less than tablet. Currently, the number of headshots per line is always changing but we want each team to have the exact layout specified in the design.
Screen Shot 2022-10-24 at 11 43 23 PM

**Review by mike10911 — CHANGES_REQUESTED** [review]
HEYY LISA!!! THIS IS LOOKING SO GOOD. A COUPLE OF DESIGN CHANGES.
### Yours
Screen Shot 2022-10-26 at 1 54 05 AM
### Design
Screen Shot 2022-10-26 at 1 53 58 AM

I think we want the names to be a bit bigger to fit the design.

**Comment by mike10911** [comment]
Screen Shot 2022-10-26 at 1 55 09 AM

"Social Outreach" should be horizontally aligned with the other team name headings

File: main-site/components/meet-the-team/mobile-team-column/MobileTeamColumn.tsx
Comment (mike10911) [inline main-site/components/meet-the-team/mobile-team-column/MobileTeamColumn.tsx]: Im not sure if when calling methods that utilize generic type, we should parametrize it like `getLeftOrRight<TeamColumnInfo>()` or leave it as is. I would clarify with @jzhzhang66

**Review by mike10911 — CHANGES_REQUESTED** [review]

File: shared-ui/lib/types.ts
Comment (mike10911) [inline shared-ui/lib/types.ts:56]: I think this is supposed to be in the `types.ts` of the main-site directory. I would again clarify with @jzhzhang66

**Review by mike10911 — CHANGES_REQUESTED** [review]

File: shared-ui/lib/types.ts
Comment (jhylisa666) [reply shared-ui/lib/types.ts:56]: from last year's i think the meet the team thingy is in both the live site and main site

Comment (jzhzhang66) [reply shared-ui/lib/types.ts:56]: @jhylisa666 @mike10911  Oh that's a good point! Okay I'm double checking with Sarah if it's gonna be on the live-site again - will ping you once I get a response!! 

If it is going to be on the live-site, could you please move your entire meet-the-team folder into the shared-ui components folder? (and make sure all imports are updated / the site still looks good)

If it's not going to be on the live-site, could you please move the types into the main-site types file? 

Approving but please make whichever change it ends up being before you merge!

**Review by jzhzhang66 — APPROVED** [review]

File: shared-ui/lib/types.ts
Comment (jhylisa666) [reply shared-ui/lib/types.ts:56]: Sounds good!!!

**Review by mike10911 — APPROVED** [review]
LGTM once we figure out the folder location!

**Pull request merged**
